### PR TITLE
SETI-2444 fix docker run on jenkins slaves

### PR DESCRIPTION
**Pull request by Hahihula**

### CODE REVIEW

File: Dockerfile.ruby [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM ruby:2.3-alpine
 
 MAINTAINER Tomas Korcak <korczis@gmail.com>
 
-RUN apk add --no-cache curl make gcc git g++ python linux-headers binutils-gold gnupg libstdc++ openssl cmake
+RUN apk add --no-cache curl make gcc git g++ python linux-headers binutils-gold gnupg libstdc++ openssl cmake curl-dev
```
Comment (yenkins) [inline Dockerfile.ruby]: error: Lint/Syntax: unexpected token tIDENTIFIER
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`) (rubocop)

Comment (yenkins) [inline Dockerfile.ruby]: warning: Lint/Syntax: `-' interpreted as argument prefix
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`) (rubocop)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/767/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1589/ : FAILURE in 3m 01s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1850/ : SUCCESS in 3m 12s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/29/ : SUCCESS in 7m 13s

**Comment by Hahihula** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/768/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1546/ : FAILURE in 13s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul/107/ : SUCCESS in 31m 31s

**Pull request merged**